service/version_bundles.go

```diff
@@ -14,6 +14,7 @@ import (
 	kvmv1 "github.com/giantswarm/cluster-operator/service/controller/kvm/v1"
 	kvmv2 "github.com/giantswarm/cluster-operator/service/controller/kvm/v2"
 	kvmv3 "github.com/giantswarm/cluster-operator/service/controller/kvm/v3"
+	kvmv4 "github.com/giantswarm/cluster-operator/service/controller/kvm/v4"
```

Comment:
Brainwave, what about renaming the packages and save the naming hustle here?

Reply:
Do you mean the package would be something like `service/controller/kvm/kvmv4`? I kinda like the current pattern and I think this aliasing is only needed in two places.

Reply:
This was the idea. I am fine with keeping how it is. All good.